.travis.yml

```diff
@@ -5,11 +5,13 @@ env:
     - ANDROID_HOME=/usr/local/android-sdk
     - TOOLS=${ANDROID_HOME}/tools
     - PATH=${ANDROID_HOME}:${ANDROID_HOME}/emulator:${TOOLS}:${TOOLS}/bin:${ANDROID_HOME}/platform-tools:${PATH}
-    - API=28
```

Comment:
Do you know if updating the API version will cause issues for installing on some of the phones that are currently being used?